README.md

```diff
@@ -300,29 +304,7 @@ When you add a `package.json` to dependents of `foo`, you will also need to open
 
 If a package was never on Definitely Typed, it does not need to be added to `notNeededPackages.json`.
 
-#### Linter
-
-All new packages must be linted. To lint a package, add a `tslint.json` to that package containing
-
-```js
-{
-  "extends": "dtslint/dt.json"
-}
-```
-
-This should be the only content in a finished project's `tslint.json` file. If a `tslint.json` turns rules off, this is because that hasn't been fixed yet. For example:
-
-```js
-{
-  "extends": "dtslint/dt.json",
-  "rules": {
-    // This package uses the Function type, and it will take effort to fix.
-    "ban-types": false
-  }
-}
-```
-
-(To indicate that a lint rule truly does not apply, use `// tslint:disable rule-name` or better, `//tslint:disable-next-line rule-name`.)
+#### Verifying
```

Comment:
I know that this wasn't here before, but I think that it's worth starting with a quick paragraph saying that tests are preferably in files that look like that (`test` in the name or in the path), and that these files should have the following.